types/cleave.js/react/index.d.ts

```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 import { Props } from "./props";
-
 declare var Cleave: React.ComponentClass<Props>;
-export = Cleave;
+
+export { Props as CleaveProps, InitHandler } from "./props";
+export default Cleave;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`.

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).